configure.sh

```diff
@@ -1,17 +1,26 @@
 #!/bin/bash
 
+#Add necessary paths. Also add to .bashrc
+echo "Adding library paths to ~/.bashrc"
+export PYTHONPATH=$PYTHONPATH:$(pwd)/SuperBuild/install/lib/python2.7/dist-packages
+export PYTHONPATH=$PYTHONPATH:$(pwd)/SuperBuild/src/opensfm
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(pwd)/SuperBuild/install/lib
+echo 'export PYTHONPATH=$PYTHONPATH:$(pwd)/SuperBuild/install/lib/python2.7/dist-packages' >> ~/.bashrc
+echo 'export PYTHONPATH=$PYTHONPATH:$(pwd)/SuperBuild/src/opensfm' >> ~/.bashrc
+echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(pwd)/SuperBuild/install/lib' >> ~/.bashrc
```

Comment:
1. $(pwd) in the bashrc file will print literally. Gotta use `"` not `'` (single quotes).
2. Over time the bashrc will grow with repeated lines. Let's avoid that with something like this:
```
#!/bin/bash

BASHRCPATH=$(pwd)/bachrc_test
PYPATH=$(pwd)/SuperBuild/install/lib/python2.7/dist-packages
SFMPATH=$(pwd)/SuperBuild/src/opensfm
LIBPATH=$(pwd)/SuperBuild/install/lib

PATHARRAY=($PYPATH $SFMPATH $LIBPATH)
BASEARRAY=(PYTHONPATH PYTHONPATH LD_LIBRARY_PATH)

export PYTHONPATH=$PYTHONPATH:$PYPATH:$SFMPATH
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$LIBPATH

# Make sure bashrc exists; otherwise create it
if ! [ -d $BASHRCPATH ]; then
    echo "$BASHRCPATH doesn't exist. Creating it now\n"
    touch $BASHRCPATH
fi

# create the
count=0
while [ "x${PATHARRAY[$count]}" != "x" ]
do
    if $(cat $BASHRCPATH | grep -x ${PATHARRAY[$count]}); then
        echo "Path already exists; skipping . . . \n"
    else
        echo "export ${BASEARRAY[$count]}=\$${BASEARRAY[$count]}:${PATHARRAY[$count]}" >> $BASHRCPATH
    fi
    count=$(( $count +1 ))
done
```